engine/schema/mysql.sql

```diff
@@ -58,60 +79,30 @@ CREATE TABLE `prefix_config` (
   PRIMARY KEY (`name`,`site_guid`)
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8;
 
--- application specific configuration
-CREATE TABLE `prefix_datalists` (
-  `name` varchar(255) NOT NULL,
-  `value` text NOT NULL,
-  PRIMARY KEY (`name`)
-) ENGINE=MyISAM DEFAULT CHARSET=utf8;
-
 -- primary entity table
 CREATE TABLE `prefix_entities` (
   `guid` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
-  `type` enum('object','user','group','site') NOT NULL,
-  `subtype` int(11) DEFAULT NULL,
+  `type_id` int(11) NOT NULL, -- type
+  `subtype_id` int(11) DEFAULT NULL, -- type
+  `title_id` int(11) unsigned NOT NULL, -- metastring
+  `description_id` int(11) unsigned NOT NULL, -- metastring
   `owner_guid` bigint(20) unsigned NOT NULL,
   `site_guid` bigint(20) unsigned NOT NULL,
   `container_guid` bigint(20) unsigned NOT NULL,
-  `access_id` int(11) NOT NULL,
   `time_created` int(11) NOT NULL,
   `time_updated` int(11) NOT NULL,
   `last_action` int(11) NOT NULL DEFAULT '0',
```

Comment:
time_updated and last_action could be far more benefitial as metadata. They could serve as a log, where a metadata value is an action, metadata owner is an actor, and timestamp is that of metadata time created.